Patches to GNU Classpath 0.08 so that it works on Mac OSX.

@author Chris Hoffman
@date   16 March 2004

Copyright © IBM Corp. 2004

$Id$


diff -ru ./native/fdlibm/ieeefp.h /exp/dodo/ali/home/hoffmann/share/classpath-0.08-test1/classpath/native/fdlibm/ieeefp.h
--- ./native/fdlibm/ieeefp.h	Fri Jan 10 11:49:15 2003
+++ /exp/dodo/ali/home/hoffmann/share/classpath-0.08-test1/classpath/native/fdlibm/ieeefp.h	Wed Mar 10 11:24:23 2004
@@ -113,6 +113,11 @@
 #endif
 #endif
 
+#if (defined( __BIG_ENDIAN__) && defined( __APPLE_CC__ ))
+#define __IEEE_BIG_ENDIAN
+#endif 
+
+
 #ifdef __fr30__
 #define __IEEE_BIG_ENDIAN
 #endif
